grammars/acme.cson

```diff
@@ -33,74 +33,99 @@ repository:
   # symbols
   symbols:
     patterns: [
+#     Labels (at beginning of a line)
+      {
+        match: '^[A-Za-z][A-Za-z0-9_:]*'
```

Comment:
I suggest changing this to 
```
match: '^[A-Za-z\\._][A-Za-z0-9_:]*'
```
Global labels can start with a underscore, local labels with a colon.

Reply:
Thanks, updated! I'm just doing this PR on GoDot's behalf.